Switch/Toggle/index.jsx

```diff
@@ -86,11 +166,11 @@ const Toggle = React.createClass({
       ...remainingProps
     } = this.props
 
-    const { pressed } = this.state
+    const { pressed, pseudoValue } = this.state
 
     const classNames = classNamesBind.bind({ ...defaultStyles, ...styles })
     const cls = classNames(baseClass, {
-      'is-checked': value,
+      'is-checked': pseudoValue != null ? pseudoValue : value,
```

Comment:
The value would be `undefined`, which wouldn't match. `!=` matches both `null` and `undefined`. However, there shouldn't be a scenario where the value is `null`, so we might as well do `!== undefined`.